object.js

```diff
@@ -0,0 +1,145 @@
+const data = {
+  projects: [
+    {
+      key: 0,
+      title: "Multi-Post Stories ",
+      image: "./image/Snapshoot portfolio.png",
```

Comment:
- [ ] Maybe I just found the problem with the images. If you use a blank space between words on the file name you could have trouble, try using `Snapshoot_portoglio.png` Instead (don't forget to change the image name too).
![image](https://user-images.githubusercontent.com/20563252/123460503-565e2100-d5ad-11eb-9d9a-9ee55c23ff5f.png)